### PR TITLE
[Fix] Fix & Format pare config

### PR DESCRIPTION
**Pull request by ykk648**
1. add `img_res = 224` to fix pare model infer error
2. format pare config reference [spin/resnet50_spin_pw3d.py](https://github.com/open-mmlab/mmhuman3d/blob/main/configs/spin/resnet50_spin_pw3d.py)

#172

### CODE REVIEW

File: configs/pare/hrnet_w32_conv_pare_coco.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-# optimizer
 
+img_res = 224
+
+# optimizer
 optimizer = dict(
-    backbone=dict(type='Adam', lr=2.0e-4),
-    head=dict(type='Adam', lr=2.0e-4),
-)
+    backbone=dict(type='Adam', lr=2.0e-4), head=dict(type='Adam', lr=2.0e-4), )
```
Comment (WYJSJTU) [inline configs/pare/hrnet_w32_conv_pare_coco.py:11]: This line is too long and might cause the lint error, the same with other configs.

File: configs/pare/hrnet_w32_conv_pare_coco_cache.py [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-# optimizer
 
+img_res = 224
+
+# optimizer
 optimizer = dict(
-    backbone=dict(type='Adam', lr=2.0e-4),
-    head=dict(type='Adam', lr=2.0e-4),
-)
+    backbone=dict(type='Adam', lr=2.0e-4), head=dict(type='Adam', lr=2.0e-4), )
```
Comment (WYJSJTU) [inline configs/pare/hrnet_w32_conv_pare_coco_cache.py:11]: This line is too long and might cause the lint error, the same with other configs.

File: configs/pare/hrnet_w32_conv_pare_mix.py [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 evaluation = dict(interval=6, metric=['pa-mpjpe', 'mpjpe'])
+
+img_res = 224
+
+# optimizer
 optimizer = dict(
-    backbone=dict(type='Adam', lr=5.0e-5),
-    head=dict(type='Adam', lr=5.0e-5),
-)
+    backbone=dict(type='Adam', lr=5.0e-5), head=dict(type='Adam', lr=5.0e-5), )
```
Comment (WYJSJTU) [inline configs/pare/hrnet_w32_conv_pare_mix.py:11]: This line is too long and might cause the lint error, the same with other configs.

File: configs/pare/hrnet_w32_conv_pare_mix_cache.py [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
 evaluation = dict(interval=6, metric=['pa-mpjpe', 'mpjpe'])
+
+img_res = 224
+
+# optimizer
 optimizer = dict(
-    backbone=dict(type='Adam', lr=5.0e-5),
-    head=dict(type='Adam', lr=5.0e-5),
-)
+    backbone=dict(type='Adam', lr=5.0e-5), head=dict(type='Adam', lr=5.0e-5), )
```
Comment (WYJSJTU) [inline configs/pare/hrnet_w32_conv_pare_mix_cache.py:11]: This line is too long and might cause the lint error, the same with other configs.

File: configs/pare/hrnet_w32_conv_pare_mix_no_mosh.py [hunk 5]
```diff
@@ -1,12 +1,16 @@
...
 evaluation = dict(interval=6, metric=['pa-mpjpe', 'mpjpe'])
+
+img_res = 224
+
+# optimizer
 optimizer = dict(
-    backbone=dict(type='Adam', lr=5.0e-5),
-    head=dict(type='Adam', lr=5.0e-5),
-)
+    backbone=dict(type='Adam', lr=5.0e-5), head=dict(type='Adam', lr=5.0e-5), )
```
Comment (WYJSJTU) [inline configs/pare/hrnet_w32_conv_pare_mix_no_mosh.py:11]: This line is too long and might cause the lint error, the same with other configs.

Comment (ttxskk) [reply configs/pare/hrnet_w32_conv_pare_mix_no_mosh.py:11]: Hi @ykk648, thanks for your contribution! You can run `pre-commit` to fix the lint error. More details can be found in [CONTRIBUTING.md](https://github.com/open-mmlab/mmhuman3d/blob/main/.github/CONTRIBUTING.md).

**Review by ttxskk — APPROVED** [review]

**Pull request merged**